src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -144,3 +145,23 @@ export const getLocalDateTimeForGuid = (
     .toFormat('DDD')
     .concat(', ', localDateTime.toFormat('t'));
 };
+
+/**
+ * Converts an user's Mozilla ldap username e.g ad|Mozilla-LDAP|jdoe to jdoe
+ * @param ldapString
+ * @returns Curator's username in the following format: flastname
+ */
+export const getCuratorNameFromLdap = (ldapString: string) =>
+  ldapString.split('|')[2];
+
+/**
+ * Converts a scheduled surface guid e.g "NEW_TAB_EN_US" to a readable format
+ * @param surfaceGuid
+ * @returns Readable scheduled surface name e.g New Tab (en-US)
+ */
+export const getScheduledSurfaceName = (surfaceGuid: string) => {
+  return (
+    ScheduledSurfaces.find((surface) => surface.guid === surfaceGuid)?.name ??
+    'Unknown Surface'
+  );
+};
```

Comment:
Let's add tests for these straight away. Happy path + any edge cases (such as the LDAP string being empty/not formatted correctly) should not take too much time for these helper functions.